### ISSUE TITLE
Flagging analyzer experiments in build_test

### ISSUE THREAD

**Issue by shyndman** [dart-lang/build#2576]
Hi there,

A user of a library I contribute to mentioned that the code that we generate is invalid under NNDB (https://github.com/mobxjs/mobx.dart/issues/332), so I started investigating to get an idea of how much work it will be for us to implement.

Although I was able to find the NNDB feature flag without much trouble, I had a harder time seeing how I could enable the feature for specific `build_test` unit tests.

I know this feature is experimental, subject to change and all that. I don't expect a solution quickly, but I thought you might be able to shed some light in case there was a way to do this.

**Comment by jakemac53** [comment]
I think this is a duplicate of https://github.com/dart-lang/build/issues/2343

**Comment by jakemac53** [comment]
We are currently blocked on analyzer features but that should provide some insight into how this will eventually be done

**Comment by shyndman** [comment]
Excellent. Thank you.
